cdap-cli/src/main/java/co/cask/cdap/cli/command/DeleteDatasetInstanceCommand.java

```diff
@@ -43,10 +44,11 @@ public DeleteDatasetInstanceCommand(DatasetClient datasetClient, CLIConfig cliCo
 
   @Override
   public void perform(Arguments arguments, PrintStream output) throws Exception {
-    String datasetName = arguments.get(ArgumentName.DATASET.toString());
+    Id.DatasetInstance dataset = Id.DatasetInstance.from(cliConfig.getCurrentNamespace(),
+                                                         arguments.get(ArgumentName.DATASET.toString()));
 
-    datasetClient.delete(datasetName);
-    output.printf("Successfully deleted dataset named '%s'\n", datasetName);
+    datasetClient.delete(dataset);
+    output.printf("Successfully deleted %s\n", dataset);
```

Comment:
we don't have to worry that the output format of some CLI commands now changes, right? Do we worry about compatibility there?